compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1475,9 +1475,6 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsArraySet() {return _flags1.testAny(SupportsArraySet);}
    void setSupportsArraySet() {_flags1.set(SupportsArraySet);}
 
-   bool getSupportsArraySetToZero() {return _flags3.testAny(SupportsArraySetToZero);}
```

Comment:
Since supporting arrayset to zero is technically a 'sub-feature' of arrayset, we need to ensure that if a CG declares supporting for arbitrary arrayset, it must also declare supporting for arrayset to zero; otherwise, it may lead to confusion.